cg/apps/demultiplex/notes.md

```diff
@@ -0,0 +1,100 @@
+We loop over all flowcell directories and check if sequencing and transfering is done. This is indicated by the two
+files RTAComplete.txt and CopyComplete.txt.
+We also create file (demuxcomplete.txt) that indicates if is finished.
+
+## Output to
+
+`/home/proj/stage/demultiplexed-runs/`
+
+## WHY DO WE NEED TO MAKE A DIFFERENCE BETWEEN FLUFFY AND WGS
+
+    - we don't want to add dummy samples to fluffy sample sheet
+    - We don't want to pad the fluffy indexes
+
+
+Distinction can be made from that the 
+    
+    - index length for fluffy is 8 and wgs 10
+    - read length for fluffy is 51 and 151 for wgs
+
+### THIS IS THE PART I DON'T UNDERSTAND ATM
+
+- I think that the file delivery.txt indicates that demultiplexing has started.
+
+
+- Then check if there is a SampleSheet.csv file, if not this is created by fetching raw information from LIMS and then
+convert it to a format that bcl2fastq accepts.
+
+- When that file exists we start demultiplexing by sending a batch job with instructions to slurm.
+
+- Before starting demux we will need understand what basemask to use. Basemask is a value derived from the runparameters 
+file. It has to do with the read length and the index length.
```

Comment:
Correct, and it is also depended on the indexes in the sample sheet, if the index lengths differ from the runparameters. Not needed for novaseq.

Reply:
👍